src/commands/helm-conftest.yaml

```diff
@@ -0,0 +1,13 @@
+---
+parameters:
+  chart:
+    type: "string"
+description: Test conftest policies
+steps:
+  - run: |
+      if [ -d "helm/<<parameters.chart>>/ci" ]; then
+        for t in $(ls helm/<<parameters.chart>>/ci/*.yaml); do helm template --values $t helm/<<parameters.chart>> | conftest test --update https://raw.githubusercontent.com/swade1987/deprek8ion/master/1.16-deprek8ion.rego,https://raw.githubusercontent.com/swade1987/deprek8ion/master/1.17-deprek8ion.rego,https://raw.githubusercontent.com/swade1987/deprek8ion/master/1.18-deprek8ion.rego,https://raw.githubusercontent.com/swade1987/deprek8ion/master/1.19-deprek8ion.rego azure-operator -; done
```

Comment:
We have `azure-operator` in here. I also wonder if we can get away w/o this `if`.

Reply:
Typo, fixing it.